CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   * Fix JS command `:target` failing to work when used as phx-submit or phx-change with a selector-based target
 
 ### Enhancements
-  * Speed up DOM patching performance 3-30x 🔥
```

Comment:
The 🔥 didn't seem sufficient. Feel free to close if you're not as pumped as I am.